base/linalg/qr.jl

```diff
@@ -1,14 +1,50 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
 # QR and Hessenberg Factorizations
+"""
+    QR <: Factorization
+
+Represents a QR factorization of an `m`×`n`  matrix `A`, with the `Q` matrix stored as a sequence of Householder reflectors:
```

Comment:
Should "Represents" be "Represent" for consistency with other documentation? Or does that guideline not apply to docstrings for types?

Also an extra space after the last ``` ` ``` in ``` an `m`x`n`  matrix```?